mw-webapp/src/dataAccessLogic/MentorCommentDAL.ts

```diff
@@ -24,25 +25,33 @@ export class MentorCommentDAL {
   /**
    * Create MentorComment
    */
-  public static async createMentorComment(): Promise<MentorComment> {
+  public static async createMentorComment(dayReportUuid: string): Promise<MentorComment> {
     const mentorCommentWithoutUuid: MentorCommentDTOWithoutUuid = {
-      description: SPACE,
+      description: UnicodeSymbols.ZERO_WIDTH_SPACE,
       mentorUuid: "",
       isDone: false,
     };
 
     const newMentorComment = await MentorCommentService.createMentorCommentDTO(mentorCommentWithoutUuid);
 
     const mentorComment = mentorCommentDTOToMentorCommentConverter(newMentorComment);
+    const updatedDayReport = await DayReportDAL.getDayReport(dayReportUuid);
+    const updatedMentorComment = [...updatedDayReport.mentorComments, mentorComment];
+    const dayReportUpdated = {...updatedDayReport, mentorComments: updatedMentorComment};
+    await DayReportDAL.updateDayReport(dayReportUpdated);
```

Comment:
I see three requests in a row one by one in such files, is it ok? Could we optimize this part?

Reply:
Deleted one request

Reply:
Did you check other places? Shuld it be optimized in other task
?